opensearch-ruby.gemspec

```diff
@@ -66,4 +66,5 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'faraday', '>= 1.0', '< 3'
   s.add_dependency 'multi_json', '>= 1.0'
+  s.add_dependency 'base64'
```

Comment:
Linter flagged this because the dependencies are supposed to be listed in alphabetical order. Would you mind moving this before `multi_json`?

Reply:
I fixed the order with `rubocop -a`
https://github.com/opensearch-project/opensearch-ruby/pull/218/commits/4ec2e8ed0720cbb2a836cb554403e4235fab5811